### PR TITLE
Sinc dvr

### PR DESCRIPTION
**Pull request by halvarsu**
Allows basis change to for example a HF-basis. Overwrites sparsity in new basis.

I was a bit unsure how to do this, as `transform_two_body_elements` is originally a `@staticmethod`. This works by overwriting with a `@classmethod`.

### CODE REVIEW

File: tests/test_one_dim_qd.py [hunk 1]
```diff
@@ -2,7 +2,11 @@
 import pytest
 import numpy as np
 
-from quantum_systems import ODQD, GeneralOrbitalSystem, SpatialOrbitalSystem
+from quantum_systems import (
+    ODSincDVR,
```
Comment (Schoyen) [inline tests/test_one_dim_qd.py]: This needs to be corrected to `ODQD` again.

Comment (halvarsu) [reply tests/test_one_dim_qd.py]: oh haha

**Comment by halvarsu** [comment]
BTW, there are more changes incoming, for example fock matrix calculations needs to be fixed

**Comment by Schoyen** [comment]
Keep 'em comin'!

**Pull request merged**